roles/telemetry_logging/tasks/main.yml

```diff
@@ -0,0 +1,500 @@
+- name: Prepare Logging Tests
+  hosts: localhost
+  connection: local
+  gather_facts: false
+# vars:
+#   ansible_user: "{{ lookup('env', 'USER') }}"      
+#   APATH: "{{ lookup('env', 'PATH') }}"
+#   AHOME: "{{ lookup('env', 'HOME') }}"
+
+  #environment:
+  #  PATH: "{{ APATH }}:{{ AHOME }}/.local/bin/:/usr/local/bin/"
+
+  tasks:
+    - name: "Log into OCP"
+      ansible.builtin.shell:
+        cmd: |
+          oc login -u kubeadmin -p "12345678" https://api.crc.testing:6443
+      failed_when: false
+      changed_when: false
+
+
+- name: 'Verify projects exist'
+  hosts: localhost
```

Comment:
So, this main.yml should not be a series of plays, and it should not reference hosts.

However, the role is going to be imported in a playbook somewhere, and the hosts, etc will have to be configured.

Typically, I think "controller" is used for localhost.

When running on Zuul, `localhost` refers to the zuul executer node, which we don't want to run stuff on in general, since it's not part of the test environment. 

There are some other nodes listed in this file as well, and I'll add some notes to the relevant lines below